qcodes/instrument_drivers/stanford_research/SR830.py

```diff
@@ -3,13 +3,73 @@
 from typing import Any
 
 from qcodes import VisaInstrument
-from qcodes.instrument.parameter import ArrayParameter, ParamRawDataType
-from qcodes.utils.validators import Numbers, Ints, Enum, Strings
+from qcodes.instrument.parameter import ArrayParameter, ParamRawDataType, ParameterWithSetpoints, Parameter
+from qcodes.utils.validators import Numbers, Ints, Enum, Strings, Arrays
 
 from typing import Tuple
 import time
 
 
+class ChannelTrace(ParameterWithSetpoints):
+    """
+    Parameter class for the two channel buffers
+    """
+
+    def __init__(self, name: str, channel: int, **kwargs) -> None:
```

Comment:
To fix the mypy error here you should type **kwargs as `Any`

Reply:
done thanks